linux_os/guide/services/ssh/ssh_server/sshd_disable_pubkey_auth/rule.yml

```
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+title: 'Disable PubkeyAuthentication Authentication'
+
+description: |-
+    Unless needed, SSH should not permit extraneous or unnecessary
+    authentication mechanisms. To disable PubkeyAuthentication authentication, add or
+    correct the following line in the <tt>/etc/ssh/sshd_config</tt> file:
+    <pre>PubkeyAuthentication no</pre>
+
```

```suggestion
severity: medium
```
It's also required to include severity of the rule, you can find more information regarding severity here: https://github.com/ComplianceAsCode/content/blob/master/docs/manual/developer_guide.adoc#711-rules

---

I'll get that merged into my next rebase.